src/TTTSelector.cc

```diff
@@ -0,0 +1,443 @@
+#include "Analysis/VVAnalysis/interface/TTTSelector.h"
+#include "TLorentzVector.h"
+#include <TStyle.h>
+#include <regex>
+#include "TParameter.h"
+
+#define Fill1D(NAME, VALUE_) HistFullFill(histMap1D_, NAME, variation.second, VALUE_, weight);
```

Comment:
Your code your rules by I think macros are kinda gross. :)

Reply:
Meh! I love me a macro every once and a while... I like my code terse! :stuck_out_tongue_closed_eyes: